lblod/frontend-reglementaire-bijlage
# GN-4330: allow single/multiselect specification per codelist instance

-------

## Overview


Updates the plugins so it can use the new codelist single/multi select option
also removes those options from the codelist management page.

No other changes were needed cause everything is backwards compatible
(if a codelist has no explicit style, the backend will be polled).

##### connected issues and PRs:


[GN-4330](https://binnenland.atlassian.net/browse/GN-4330)
[backend pr](https://github.com/lblod/app-reglementaire-bijlage/pull/68)
### Setup



### How to test/reproduce


make codelist with qa env, proxy this pr to qa env, verify everything still works
optionally add `<VariablePlugin::Codelist::Edit @controller={{this.editor}}/>` to
`app/templates/regulatory-attachments/edit.hbs` temporarily to verify the option select is working
as intended, or use GN.
### Challenges/uncertainties




### Checks PR readiness
